eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/ChannelOpenDualFunded.scala

```diff
@@ -455,24 +455,17 @@ trait ChannelOpenDualFunded extends DualFundingHandlers with ErrorHandlers {
               stay() using d.copy(rbfStatus = RbfStatus.RbfAborted) sending TxAbort(d.channelId, InvalidPushAmount(d.channelId, d.remotePushAmount, msg.fundingContribution.toMilliSatoshi).getMessage)
             } else {
               log.info("our peer wants to raise the feerate of the funding transaction (previous={} target={})", d.fundingParams.targetFeerate, msg.feerate)
-              val fundingParams = InteractiveTxParams(
-                d.channelId,
-                d.fundingParams.isInitiator,
-                d.fundingParams.localAmount, // we don't change our funding contribution
-                msg.fundingContribution,
-                d.fundingParams.fundingPubkeyScript,
-                msg.lockTime,
-                d.fundingParams.dustLimit,
-                msg.feerate,
-                d.fundingParams.requireConfirmedInputs,
+              val fundingParams = d.fundingParams.copy(
+                // we don't change our funding contribution
+                remoteAmount = msg.fundingContribution,
+                lockTime = msg.lockTime,
+                targetFeerate = msg.feerate
```

Comment:
This makes it clearer what parameters are actually modified.

Reply:
But it's also a bit dangerous because we may forget to update some of the parameters...having them all listed ensures we think about each one and whether it can still use the previous value or not. I don't have a strong opinion, both approaches have their pros and cons.

Reply:
I prefer the `copy`, as it is more friendly for casual readers. Indeed special care will be needed for future parameters, but I trust our review process!